components/Article.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState, useEffect } from "react";
+import Moment from "react-moment";
+import Seo from "./strapi/seo";
+import Image from "next/image";
+import Link from "next/link";
+import Footer from "../components/Footer";
+import styles from "../styles/Article.module.css";
+import sanitizeHtml from "sanitize-html";
+import he from "he";
+import { formatSeo } from "../utils";
+import { Typography, Header } from "@originprotocol/origin-storybook";
+
+const sanitizationOptions = {
+  allowedTags: [
+    "b",
+    "i",
+    "em",
+    "strong",
+    "u",
+    "a",
+    "img",
+    "h1",
+    "h2",
+    "h3",
+    "span",
+    "p",
+    "ul",
+    "ol",
+    "li",
+    "br",
+    "figure",
+  ],
+  allowedAttributes: {
+    a: ["href", "target", "rel"],
+    img: ["src", "alt", "srcset", "sizes"],
+    span: ["style"],
+    ul: ["style"],
+    ol: ["style"],
+  },
+  allowedIframeHostnames: ["www.youtube.com"],
+};
+
+const Article = ({ article, navLinks }) => {
+  const [loaded, setLoaded] = useState(false);
+  const imageUrl = article.cover?.url;
+  const seo = formatSeo(article.seo);
+
+  useEffect(() => {
+    setLoaded(true);
+  }, []);
+
+  return (
+    <>
+      <Seo seo={seo} />
+      {loaded && (
+        <>
+          <Header mappedLinks={navLinks} webProperty="ousd" />
+          <div className="bg-[#141519] px-8 md:px-16 lg:px-[134px] pb-8 md:pb-12">
+            <div className="max-w-[763px] mx-auto">
+              <Link
+                href={"/blog"}
+                className="inline-block p-[1px] rounded-full gradient2"
+              >
+                <div className="w-full h-full px-4 md:px-6 py-1.5 text-center rounded-full bg-[#141519]">
+                  <div className="flex flex-row justify-between space-x-3 md:space-x-5">
+                    <Image
+                      src="/images/arrow-left-gradient.svg"
+                      width="10"
+                      height="6"
+                      className=""
+                      alt="arrow"
+                    />
+                    <Typography.Body3
+                      className="text-[12px] md:text-[16px]"
+                      style={{ fontWeight: 500 }}
+                    >
+                      Back to News
+                    </Typography.Body3>
+                  </div>
+                </div>
+              </Link>
+              <div className="mt-6 md:mt-12">
+                <Typography.H4
+                  as="h1"
+                  className="text-[24px] md:text-[44px] leading-[32px] md:leading-[60px]"
```

Comment:
nit: We should avoid hardcoding absolute values. Makes it harder if we have to change the sizes or colours of all things, would be a hassle